lib/main_publiccloud.pm

```diff
@@ -77,47 +77,68 @@ sub load_publiccloud_consoletests {
     loadtest 'console/libgcrypt';
 }
 
+sub load_released_publiccloud_tests {
```

Comment:
we not testing anything "released" so this name sounds confusing to me .  I open to hear other ideas . Only what I have in mind is `load_latest_publiccloud_tests` . I agree it is also not very descriptive but in any case I strongly disagree with "released"

Reply:
I agree with Anton, `latest` might be the better term